documents/Libraries/stdlib/stdlib_sx-glsl_impl.mtlx

```diff
@@ -784,13 +812,27 @@
   <!-- ======================================================================== -->
 
   <!-- <blur> -->
-  <implementation name="IM_blur_float_sx_glsl" nodedef="ND_blur_float" language="sx-glsl"/>
-  <implementation name="IM_blur_color2_sx_glsl" nodedef="ND_blur_color2" language="sx-glsl"/>
-  <implementation name="IM_blur_color3_sx_glsl" nodedef="ND_blur_color3" language="sx-glsl"/>
-  <implementation name="IM_blur_color4_sx_glsl" nodedef="ND_blur_color4" language="sx-glsl"/>
-  <implementation name="IM_blur_vector2_sx_glsl" nodedef="ND_blur_vector2" language="sx-glsl"/>
-  <implementation name="IM_blur_vector3_sx_glsl" nodedef="ND_blur_vector3" language="sx-glsl"/>
-  <implementation name="IM_blur_vector4_sx_glsl" nodedef="ND_blur_vector4" language="sx-glsl"/>
+  <implementation name="IM_blur_float_sx_glsl" nodedef="ND_blur_float" language="sx-glsl">
+      <parameter name="filtertype" type="string" implname="filtertype" impltype="integer" enumvalues="0,1"/>
```

Comment:
Having this means that we don't require a hack to map this to an int within the blur implementation node (which results in 2 uniforms vs 1 being created). The uniform for this input can be modified directly in the shader without knowing about the "hack" second uniform.